### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by TomAugspurger**
staging.binder.pangeo.io should be up and running with dask gateway. This updates the docs for when we're ready to apply things to dask-gateway.

This diff (`git diff 3b7813 deploy/staging.yaml pangeo-binder/`) gives a decent idea of the cumulative change, aside from the additional secrets.

```diff
diff --git a/deploy/staging.yaml b/deploy/staging.yaml
index 64816db..1d82ae6 100644
--- a/deploy/staging.yaml
+++ b/deploy/staging.yaml
@@ -34,6 +34,11 @@ binderhub:
     scheduling:
       userScheduler:
         enabled: false
+    singleuser:
+      extraEnv:
+        DASK_GATEWAY__ADDRESS: "https://hub.staging.binder.pangeo.io/services/dask-gateway/"
+        DASK_GATEWAY__PROXY_ADDRESS: "tls://scheduler-public-staging-dask-gateway:8786"
+
     hub:
       resources:
         requests:
@@ -42,7 +47,7 @@ binderhub:
       services:
         dask-gateway:
           # This makes the gateway available at ${HUB_URL}/services/dask-gateway
-          url: http://web-public-pangeo-binder-staging-dask-gateway
+          url: http://web-public-staging-dask-gateway
 
     proxy:
       chp:
diff --git a/pangeo-binder/requirements.yaml b/pangeo-binder/requirements.yaml
index 1e26dc8..5cf5c23 100644
--- a/pangeo-binder/requirements.yaml
+++ b/pangeo-binder/requirements.yaml
@@ -1,7 +1,7 @@
 # requirements.yaml
 dependencies:
 - name: binderhub
-  version: 0.2.0-n093.h7cfad48
+  version: 0.2.0-n132.h1a8ce62
   repository: https://jupyterhub.github.io/helm-chart/
   import-values:
     - child: rbac
@@ -12,3 +12,6 @@ dependencies:
 - name: nginx-ingress
   version: 0.20.0
   repository: https://kubernetes-charts.storage.googleapis.com
+- name: dask-gateway
+  version: "0.6.1"
+  repository: 'https://dask.org/dask-gateway-helm-repo/'
diff --git a/pangeo-binder/values.yaml b/pangeo-binder/values.yaml
index 662be7c..e63d836 100644
--- a/pangeo-binder/values.yaml
+++ b/pangeo-binder/values.yaml
@@ -4,7 +4,7 @@ binderhub:
 
   config:
     BinderHub:
-      build_image: jupyter/repo2docker:0.10.0-186.g900d280
+      build_image: jupyter/repo2docker:0.11.0-20.g070e4a9
       use_registry: true
       # use_named_servers: true  # turn back on to use auth
       # auth_enabled: true  # turn back on to use auth
@@ -59,6 +59,10 @@ binderhub:
         limit: 8G
         guarantee: 8G
       cmd: jupyter-lab
+      extraEnv:
+        DASK_GATEWAY__AUTH__TYPE: "jupyterhub"
+        DASK_GATEWAY__CLUSTER__OPTIONS__IMAGE: '{JUPYTER_IMAGE_SPEC}'
+
       # to make notebook servers aware of hub
       # cmd: jupyterhub-singleuser  # turn back on to use auth
       # start in jupyter lab
@@ -182,51 +186,54 @@ kube-lego:
   image:
     tag: 0.1.7
 
-gateway:
-  clusterManager:
-    image:  # TODO
-      name: pangeo/base-notebook
-      tag: 2019.12.24
-    clusterStartTimeout: 600
-    workerStartTimeout: 600
-    worker:
-      extraPodConfig:
-        tolerations:
-          - key: "k8s.dask.org_dedicated"
-            operator: "Equal"
-            value: "worker"
-    scheduler:
-      extraPodConfig:
-        tolerations:
-          - key: "k8s.dask.org_dedicated"
-            operator: "Equal"
-            value: "worker"
-  extraConfig:
-    # Use the mapping form, to support merging multiple values.yaml
-    optionHandler: |
-      from dask_gateway_server.options import Options, Integer, Float, String
+dask-gateway:
+  gateway:
+    loglevel: DEBUG
+    clusterManager:
+      image:
+        name: daskgateway/dask-gateway-server
+        tag: 0.6.1
+        pullPolicy: IfNotPresent
+      clusterStartTimeout: 600
+      workerStartTimeout: 600
+      worker:
+        extraPodConfig:
+          tolerations:
+            - key: "k8s.dask.org_dedicated"
+              operator: "Equal"
+              value: "worker"
+      scheduler:
+        extraPodConfig:
+          tolerations:
+            - key: "k8s.dask.org_dedicated"
+              operator: "Equal"
+              value: "worker"
+    extraConfig:
+      # Use the mapping form, to support merging multiple values.yaml
+      optionHandler: |
+        from dask_gateway_server.options import Options, Integer, Float, String
 
-      def option_handler(options):
-          if ":" not in options.image:
-              raise ValueError("When specifying an image you must also provide a tag")
-          return {
-              "worker_cores_limit": options.worker_cores,
-              "worker_cores": min(options.worker_cores / 2, 1),
-              "worker_memory": "%fG" % options.worker_memory,
-              "image": options.image,
-          }
-      c.DaskGateway.cluster_manager_options = Options(
-          Integer("worker_cores", 2, min=1, max=4, label="Worker Cores"),
-          Float("worker_memory", 4, min=1, max=8, label="Worker Memory (GiB)"),
-          String("image", default="pangeo/base-notebook:2019.12.24", label="Image"),
-          handler=option_handler,
-      )
+        def option_handler(options):
+            if ":" not in options.image:
+                raise ValueError("When specifying an image you must also provide a tag")
+            return {
+                "worker_cores_limit": options.worker_cores,
+                "worker_cores": min(options.worker_cores / 2, 1),
+                "worker_memory": "%fG" % options.worker_memory,
+                "image": options.image,
+            }
+        c.DaskGateway.cluster_manager_options = Options(
+            Integer("worker_cores", 2, min=1, max=4, label="Worker Cores"),
+            Float("worker_memory", 4, min=1, max=8, label="Worker Memory (GiB)"),
+            String("image", default="pangeo/base-notebook:2019.12.24", label="Image"),
+            handler=option_handler,
+        )
 
-    userLimts: |
-      c.UserLimits.max_cores = 100
-      c.UserLimits.max_memory = "400 G"
-      c.UserLimits.max_clusters = 1
+      userLimts: |
+        c.UserLimits.max_cores = 100
+        c.UserLimits.max_memory = "400 G"
+        c.UserLimits.max_clusters = 1
 
-    tlsURL: |
-      if isinstance(c.DaskGateway.public_connect_url, str):
-          c.DaskGateway.public_connect_url += "/services/dask-gateway"
+      tlsURL: |
+        if isinstance(c.DaskGateway.public_connect_url, str):
+            c.DaskGateway.public_connect_url += "/services/dask-gateway"

```

cc @jhamman

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -47,60 +47,43 @@ After running the cookiecutter command, simply follow the command line instructi
...
-.. .dask/config.yaml
+.. code-block:: yaml
 
-::
+   # binder/environment.yml
+   channels:
+     - conda-forge
+   dependencies:
+     - dask-gateway
+     - dask-labextension
```
Comment (jhamman) [inline docs/index.rst]: perhaps we just point to the `pangeo-notebook` metapackage?

cc @scottyhq

Comment (TomAugspurger) [reply docs/index.rst]: Seems reasonable, I'll try that out.

**Comment by TomAugspurger** [comment]
Using pangeo-notebook seems to work well. Thanks.

I think we can plan on merging this once staging is merged to prod, which I can do today.

**Review by jhamman — APPROVED** [review]

**Comment by jhamman** [comment]
I think you can merge this to staging now since the staging/prod branches are built separately on read the docs.

**Pull request merged**